pypesto/sampling/sample.py

```diff
@@ -68,6 +68,8 @@ def sample(
     sampler.sample(n_samples=n_samples)
     t_elapsed = process_time() - t_start
 
+    logger.warning("Elapsed time: "+str(t_elapsed))
```

Comment:
It may be nice to plot this, but I would not have it as a warning. maybe `logger.info` for the moment, and then we think about what information we really want to print.

Reply:
I agree, however logger.info does not print the message in the python notebook. This is why I went for `warning` instead... any clue about this?

- Plotting how? What is your suggestion?

Reply:
plot = print, sorry. yes, but warning should be confined to things were we really nice to warn the user. we may soon decide to lift the default pypesto log level to info however, then such things would be shown.

Reply:
done

Reply:
see also #367 for further discussion